thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/client/cache/SchemaCacheLoader.java

```diff
@@ -43,8 +43,12 @@ public Schema load(String collection) throws Exception {
         LOGGER.error("Schema {} not found, {}", collection, res.getStatusLine().toString());
       }
       InputStream content = res.getEntity().getContent();
+
       Schema schema = new ObjectMapper().readValue(content, Schema.class);
```

Comment:
If this method is called by single thread, there is no reason to create ObjectMapper instance for every load call. 
If multiple threads are calling this method, keep ObjectMapper instance in threadlocal.